installing/installing_sno/install-sno-installing-sno.adoc

```diff
@@ -105,4 +105,31 @@ You can use dedicated or shared IFLs to assign sufficient compute resources. Res
 
 include::modules/install-sno-ibm-z.adoc[leveloffset=+2]
 
-include::modules/install-sno-ibm-z-kvm.adoc[leveloffset=+2]
\ No newline at end of file
+include::modules/install-sno-ibm-z-kvm.adoc[leveloffset=+2]
+
+== Installing {sno} with {ibmpowerProductName}
+
+Installing a single node cluster on {ibmpowerProductName} requires user-provisioned installation using either the "Installing a cluster with {op-system-base} KVM on {ibmpowerProductName}" procedure.
+
+[NOTE]
+====
+Installing a single-node cluster on {ibmpowerProductName} simplifies installation for development and test environments and requires less resource requirements at entry level.
```

Comment:
@stevsmit I have made it uniform now. Please check. Thanks.

Reply:
@alishaIBM Can you please also change it in line 81 (IBM Z entry) and in 
modules/installation-aws_con_installing-sno-on-aws.adoc

Reply:
> Sometimes you have it written as "single-node cluster" (with they hyphen), and other times you have it written as "single node cluster".
> 
> I suggest that we fix these for parallelism. I think that it should be with the hyphen but I'm not 100% sure. Like Mike, I'm confused if it's {sno} cluster, or single-node cluster. Sometimes in this document we do have {sno} cluster written. For example, in the <6> attribute in the example YAML file.

@stevsmit I think this might be something to tackle after GA together with folks who originally set up SNO docs. The mix is throughout the existing SNO docs. WDYT?

Reply:
@SNiemann15 Yeah I guess I'm fine with that so long as the SNO writers are aware of it.

Reply:
I have updated the section for IBM Z. Also updated, modules/installation-aws_con_installing-sno-on-aws.adoc.
Please tag SNO writers if needed. Thanks.
FYI: @SNiemann15 @stevsmit

Reply:
@johnwilkins Can you PTAL